src/PrestaShopBundle/Bridge/Helper/HelperListBridge.php

```diff
@@ -0,0 +1,523 @@
+            }
+        } while (empty($this->_list));
+
+        if ($helperListConfiguration->table == 'feature') {
+            $nbItems = count($helperListConfiguration->list);
+            for ($i = 0; $i < $nbItems; ++$i) {
+                $item = &$helperListConfiguration->list[$i];
+
+                $query = new DbQuery();
+                $query->select('COUNT(fv.id_feature_value) as count_values');
+                $query->from('feature_value', 'fv');
+                $query->where('fv.id_feature =' . (int) $item['id_feature']);
+                $query->where('(fv.custom=0 OR fv.custom IS NULL)');
+                $res = Db::getInstance(_PS_USE_SQL_SLAVE_)->getValue($query);
+                $item['value'] = (int) $res;
+                unset($query);
+            }
+        }
+
+        $this->hookDispatcher->dispatchWithParameters('action' . $helperListConfiguration->controllerNameLegacy . 'ListingResultsModifier', [
+            'list' => &$helperListConfiguration->list,
+            'list_total' => &$helperListConfiguration->listTotal,
+        ]);
+    }
+
+    private function getUser(TokenStorageInterface $tokenStorage)
+    {
+        if (null === $token = $tokenStorage->getToken()) {
+            return null;
+        }
+
+        if (!\is_object($user = $token->getUser())) {
+            return null;
+        }
+
+        if (!$user instanceof Employee) {
+            return null;
+        }
+
+        return $user;
+    }
+
+    /**
+     * @param int $limit
+     *
+     * @return int
+     */
+    private function checkSqlLimit(HelperListConfiguration $helperListConfiguration, $limit)
+    {
+        if (empty($limit)) {
+            if (
+                isset($this->context->cookie->{$helperListConfiguration->listId . '_pagination'}) &&
+                $this->context->cookie->{$helperListConfiguration->listId . '_pagination'}
+            ) {
+                $limit = $this->context->cookie->{$helperListConfiguration->listId . '_pagination'};
+            } else {
+                $limit = $helperListConfiguration->defaultPagination;
+            }
+        }
+
+        $limit = (int) Tools::getValue($helperListConfiguration->listId . '_pagination', $limit);
+        if (in_array($limit, $helperListConfiguration->pagination) && $limit != $helperListConfiguration->defaultPagination) {
+            $this->context->cookie->{$helperListConfiguration->listId . '_pagination'} = $limit;
+        } else {
+            unset($this->context->cookie->{$helperListConfiguration->listId . '_pagination'});
+        }
+
+        if (!is_numeric($limit)) {
+            throw new PrestaShopException('Invalid limit. It should be a numeric.');
+        }
+
+        return $limit;
+    }
+
+    /**
+     * @return string
+     */
+    private function getFromClause(HelperListConfiguration $helperListConfiguration)
+    {
+        $sql_table = $helperListConfiguration->table == 'order' ? 'orders' : $helperListConfiguration->table;
```

Comment:
```suggestion
        $sqlTable = $helperListConfiguration->table == 'order' ? 'orders' : $helperListConfiguration->table;
```

it's a minor suggestion but I think we could use `pascalCase` for PHP